scripts/water-model/water-script.py

```diff
@@ -0,0 +1,203 @@
+import pandas as pd
```

Comment:
Add a brief description of the script's purpose:

```suggestion
"""Identify dependencies on elements of the water infrastructure system
- link population (number of peopple) to potable supply systems
- link agriculture (number of acres) to irrigation systems
"""
import pandas as pd
```